### PR TITLE
Bug 1102340 - + symbol is shown on the wrong side of phone numbers +autoland

### PR DESCRIPTION
**Pull request by anefzaoui**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8f9f1cb8ebea)

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5c2f138e2e39)

File: apps/communications/dialer/test/unit/call_log_test.js [hunk 1]
```diff
@@ -331,7 +331,8 @@ suite('dialer/call_log', function() {
             (group.emergency ? 'emergencyNumber' : '');
         assert.equal(primaryInfoMain.getAttribute('data-l10n-id'), expected);
       } else {
-        assert.equal(primaryInfoMain.innerHTML, group.number);
+        assert.equal(primaryInfoMain.querySelector('bdi').innerHTML,
+              group.number);
```
Comment (DouglasSherk) [inline apps/communications/dialer/test/unit/call_log_test.js]: Indentation here is off. Please write this as:
```js
assert.equal(
  primaryInfoMain.querySelector('bdi').innerHTML, group.number);
```

**Comment by try-server-hook** [comment]
[anefzaoui Ahmed Nefzaoui (anefzaoui)](https://github.com/anefzaoui) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6e5bb19637d3)

**Comment by autolander** [comment]
The pull request could not be applied to the integration branch. Please try again after current integration is complete.

**Pull request merged**